personal-practice/Dice.java

```diff
@@ -0,0 +1,23 @@
+import java.awt.*;
```

Comment:
Avoid [wildcard imports.](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
thank you for looking at and commenting on my java files, you didnt have to do that! :)